Makefile

```diff
@@ -524,6 +527,7 @@ iso: iso.clean-repos iso.copy-rpms iso.copy-template-rpms
 	fi
 	@MAKE_TARGET="iso QUBES_RELEASE=$(QUBES_RELEASE)" ./scripts/build $(DIST_DOM0) $(INSTALLER_COMPONENT) root || exit 1
 	@ln -f $(SRC_DIR)/$(INSTALLER_COMPONENT)/build/ISO/qubes-x86_64/iso/*.iso iso/ || exit 1
+	@ln -f $(SRC_DIR)/$(INSTALLER_COMPONENT)/build/ISO/qubes-x86_64/iso/build_latest iso/ || exit 1
```

Comment:
You need to add this file to `iso/.gitignore`.

Reply:
Oh yes I added earlier but it was useless but then removed. I added it again.